src/test/kotlin/testsupport/gradleTestTemplate.kt

```diff
@@ -91,7 +88,7 @@ private data class GradleProjectInvocationContext(
 ) : TestTemplateInvocationContext {
 
   override fun getDisplayName(invocationIndex: Int): String  = "[Gradle " +
-    if (version == GradleVersion.current()) {
```

Comment:
`==` checks for structural equality and gets compiled down to (basically) `.equals` anyways (see https://kotlinlang.org/docs/reference/equality.html#structural-equality). I think `==` is the right kotlin style, so i'd like to stick with it unless im missing something

would you mind changing it back to `==`?

Reply:
When I use `==` the compiler fails:
`No method 'equals(Any?): Boolean' available`
But I honestly have no clue why. First I thought it was a null safety problem but even with `version!! == GradleVersion.current()` it is the same problem...

Reply:
ok strange - I'll merge and then figure it out